reference-app/app.js

```diff
@@ -51,7 +51,9 @@ mainModule.controller('MainCtrl', ['$scope', '$http', '$q', 'remote', 'config',
             $scope.loading = true;
             getHubData(hub).then(platforms => {
                 $scope.selectedHub = hub;
-                $scope.selectPlatform(platforms[0]);
+                $scope.selectPlatform(platforms.platforms[0]);
+                // Log any failed translators
+                logInfo(JSON.stringify(platforms.errors, null, 2));
```

Comment:
Reference app only currently supports info and error states.  We can add warning, but I don't think it's needed at the moment.

---
In reply to: [107044861](https://github.com/openT2T/opent2t-cli/pull/30#discussion_r107044861) [](ancestors = 107044861)